docs/components/Code/index.tsx

```diff
@@ -69,37 +70,39 @@ export function Code({ children }: ChildrenProps) {
   }, [framework, renderedFrameworks])
 
   return (
-    <Tabs
-      storageKey={AUTHJS_TAB_KEY}
-      items={Object.values(renderedFrameworks)}
-      selectedIndex={tabIndex}
-    >
-      {Object.keys(renderedFrameworks).map((f) => {
-        // @ts-expect-error: Hacky dynamic child wrangling
-        const child = childs.find((c) => c?.type?.name === f)
-
-        // @ts-expect-error: Hacky dynamic child wrangling
-        return Object.keys(child?.props ?? {}).length ? (
-          child
-        ) : (
-          <Tabs.Tab key={f}>
-            <p className="p-6 font-semibold rounded-lg bg-slate-100 dark:bg-neutral-950">
-              {renderedFrameworks[f]} not documented yet. Help us by
-              contributing{" "}
-              <a
-                className="underline"
-                target="_blank"
-                href={`${project.link}/edit/main/docs/pages${router.pathname}.mdx`}
-                rel="noreferrer"
-              >
-                here
-              </a>
-              .
-            </p>
-          </Tabs.Tab>
-        )
-      })}
-    </Tabs>
+    <div className={styles["tablist-fix"]}>
```

Comment:
Nextra `Tabs` does not accept `className` 😪 , not sure this is doable with Tailwind as Nextra `Tabs` does not give access to the `tablist` element, you just pass an array of labels...

Reply:
There's always a hacky way to do it with tailwind :joy: